### ISSUE TITLE
DeepSleep

### ISSUE THREAD

**Issue by nimasaj** [skoky/MKRFox1200#1]
Thanks for sharing.
As I can see in line 51, you're making a delay() rather than using LowPower.sleep(). 
Did you also have difficulties in waking up the module after deep sleep?

**Comment by skoky** [comment]
Yes, I did. But I also had a problem with coverage and I did not know if device did not wake up or coverage was too low to send data. The coverage has improved now, I will retry and let you know

**Comment by skoky** [comment]
I tested with deep sleep and it seems to work as it is not in repo. Is there any more "deep" sleep to save battery live?

**Comment by nimasaj** [comment]
I'm not sure for the code part, maybe you have to look for your components and electronics part to save more battery. If you're using voltage regulators, look for a low dropout and low quiescent current. Also, try to switch off your sensors during sleep mode.
